### DISCUSSION TITLE
No way to turn of github footer link on readthedocs

### DISCUSSION THREAD

**Discussion by trichter** [pradyunsg/furo#838] [Potential Issue]
Hi! Thanks for this nice theme!

I think the logic in page.html https://github.com/pradyunsg/furo/blob/fd3e7b94e9959870aa3c8efb221de66be923421d/src/furo/theme/furo/page.html#L185 does not allow to turn of the github logo in the footer on readthedocs. Would be nice if this was possible, e.g. with `footer_icons=[]`.

**Comment by trichter** [comment]
Related, the github and RTD footer icons do not appear with the new RTD "addon" setup.

**Reply by nineteendo** [reply]
I was able to get it working: https://github.com/nineteendo/jsonyx/blob/main/docs/source/conf.py
